conda/core/solve.py

```diff
@@ -1284,21 +1284,25 @@ def _configure_solver_for_install(self,
 
         # Configure jobs
 
-        # 1. Lock currently installed ones, if requested
-        if update_modifier is UpdateModifier.FREEZE_INSTALLED:
-            solver.add_jobs([p for p in prefix_data.package_records], api.SOLVER_LOCK)
-
-        # 2. Install/update user requested packages
+        # 1. Install/update user requested packages
         tasks = self._collect_specs_to_add(state,
                                            update_modifier=update_modifier,
                                            deps_modifier=deps_modifier,
                                            ignore_pinned=ignore_pinned,
                                            force_remove=force_remove,
                                            force_reinstall=force_reinstall)
+        explicitly_or_implicitly_requested_names = set()
```

Comment:
Nice variable name ;) We could also just call it `requested_names` and leave a comment that they can be requested both ways. 😆